.github/workflows/pythonpublish.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           # https://docs.github.com/en/packages/learn-github-packages/publishing-a-package
           username: "${{ secrets.FLYTE_BOT_USERNAME }}"
           password: "${{ secrets.FLYTE_BOT_PAT }}"
-          image_name: ${{ github.repository_owner }}/flytekit
+          image_name: ${{ github.repository_owner }}/flytekit-py37
```

Comment:
Can we keep the name of the image the same but add this as a prefix to the version?

Reply:
yes that's a good idea